sdk/tests/core/test_az_mqtt5_rpc_server_codec.c

```diff
@@ -221,6 +222,7 @@ static void az_mqtt5_rpc_server_codec_get_subscribe_topic_default_endpoint_buffe
     void** state)
 {
   (void)state;
+  az_span test_custom_sub_topic = AZ_SPAN_FROM_STR(TEST_CUSTOM_SUBSCRIPTION_TOPIC_3);
```

Comment:
Where is the corresponding format for this being passed in in this test?

Reply:
That was the wrong macro to use, good catch!